src/libstd/env.rs

```diff
@@ -986,6 +987,11 @@ mod arch {
     pub const ARCH: &str = "aarch64";
 }
 
+#[cfg(target_arch = "avr")]
+mod arch {
+    pub const ARCH: &'static str = "avr";
+}
```

Comment:
You can build `libstd` on AVR? I thought this was a `#![no_std]`-only target.

Reply:
Yeah, this was probably from the initial experimentation where we grepped wildly around the code base and made arbitrary changes as we learned. 

I’d expect this can be removed.

Reply:
Good catch, have removed.